app/src/main/java/com/android/shelfLife/ui/camera/CameraView.kt

```diff
@@ -111,25 +222,38 @@ fun startCamera(context: Context, previewView: PreviewView) {
 }
 
 @Composable
-fun PermissionDeniedScreen() {
+fun PermissionDeniedScreen(navigationActions: NavigationActions) {
   val context = LocalContext.current
-
-  Column(
-      modifier = Modifier.fillMaxSize(),
-      verticalArrangement = Arrangement.Center,
-      horizontalAlignment = Alignment.CenterHorizontally) {
-        Text(text = "Camera permission is required to scan barcodes.")
-        Spacer(modifier = Modifier.height(16.dp))
-        Button(
-            onClick = {
-              // Open app settings
-              val intent =
-                  Intent(Settings.ACTION_APPLICATION_DETAILS_SETTINGS).apply {
-                    data = Uri.fromParts("package", context.packageName, null)
+  Scaffold(
+      bottomBar = {
+        BottomNavigationMenu(
+            onTabSelect = { selected -> navigationActions.navigateTo(selected) },
+            tabList = LIST_TOP_LEVEL_DESTINATION,
+            selectedItem = Route.SCANNER)
+      }) { paddingVals ->
+        Column(
+            modifier = Modifier.fillMaxSize().padding(paddingVals),
+            verticalArrangement = Arrangement.Center,
+            horizontalAlignment = Alignment.CenterHorizontally) {
+              Text(text = "Camera permission is required to scan barcodes.")
```

Comment:
It's a good habit to develop to avoid writing hard coded string in the code, and favor the use of String.xml resources. This will makes it easier to later [localize](https://developer.android.com/guide/topics/resources/localization) the application. It also makes the app more maintainable as it is easy to modify one text, without having to remember to modify every places that are displaying the hard coded text.